include/RAJA/operators.hxx

```diff
@@ -245,11 +245,11 @@ template <typename T>
 RAJA_HOST_DEVICE constexpr T min()
 {
   return 0;
-};
+}
 template <>
 RAJA_HOST_DEVICE constexpr int16_t min<int16_t>()
 {
-  return 0xFFFF;
+  return -32768;
```

Comment:
@tepperly thanks much for helping out. Having RAJA development evolve as a community effort is something we want. I agree with @davidbeckingsale comment about the numeric_limits approach.